shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.h

```diff
@@ -131,6 +132,37 @@ class FuchsiaExternalViewEmbedder final : public flutter::ExternalViewEmbedder {
     scenic::Material material;
   };
 
+  // Helper class for setting up an invisible rectangle to catch all input. Rejected input will then
+  // be re-injected into the scene below us.
+  class InputInterceptor {
+   public:
+    InputInterceptor(scenic::Session* session) : opacity_node_(session), shape_node_(session) {
+      opacity_node_.SetLabel("Flutter::InputInterceptor");
+      opacity_node_.SetOpacity(0.f);
+
+      // Set the shape node to capture all input. Any unwanted input will be reinjected.
+      shape_node_.SetHitTestBehavior(fuchsia::ui::gfx::HitTestBehavior::kDefault);
+      shape_node_.SetSemanticVisibility(false);
+
+  scenic::Material material(session);
```

Comment:
Looks like you need to run clang-format:  buildtools/linux-x64/clang/bin/clang-format

I just set it up in my vscode project with this line in my settings:
"C_Cpp.clang_format_path": "${workspaceFolder}/../buildtools/linux-x64/clang/bin/clang-format",

Reply:
Alternatively you can use the flutter CI scripts that will format all filetypes:

./ci/format.sh  | patch -p0
./ci/check_gn_format.py --gn-binary third_party/gn/gn --root-directory .

The clang-format suggestion is just for if you want format-on-save